### PR TITLE
default to 7 seconds for first aggregation

### PR DESCRIPTION
**Pull request by potuz**
This PR updates the default aggregation time to 7 seconds instead of 6.5. In the worst case of our fleet subscribed to all subnets we see 99 percentile at less than a second (between 500ms and 1000ms) and 80 percentile under 500ms (between 200ms and 500ms).

fo

### CODE REVIEW

**Review by terencechain — DISMISSED** [review]

**Comment by terencechain** [comment]
Removed label so we can get more than my approval before merging

**Review by terencechain — APPROVED** [review]

**Review by rkapka — APPROVED** [review]

**Pull request merged**
